bin/ch/ChakraRtInterface.cpp

```diff
@@ -23,10 +24,16 @@ ChakraRTInterface::ArgInfo* ChakraRTInterface::m_argInfo = nullptr;
 TestHooks ChakraRTInterface::m_testHooks = { 0 };
 JsAPIHooks ChakraRTInterface::m_jsApiHooks = { 0 };
 
-LPCSTR GetChakraDllName()
+LPCSTR GetChakraDllName()
+{
+	return chakraDllName;
+}
```

Comment:
There s no usage of this now. Remove it.

Reply:
@agarwal-sandeep Done. Thanks!